types/react-router/index.d.ts

```diff
@@ -58,11 +59,11 @@ export interface RedirectProps {
 }
 export class Redirect extends React.Component<RedirectProps, any> { }
 
-export interface RouteComponentProps<P> {
-  match: match<P>;
-  location: H.Location;
+export interface RouteComponentProps<P, C extends {} = {}> {
```

Comment:
There's already a `StaticContext` interface defined in `test/examples-from-react-router-website/StaticRouter.tsx`. Is there a connection between the two? Should static context have `statusCode` as an optional property? If that's the same as what you have, can you integrate that interface into your PR?

Reply:
Done